tests/utils/test_json_dumps.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
Move it to `tests/test_simpleflow/utils/test_json_dumps.py` ? I'd like to switch to this new convention globally if you're OK, I started in #142 and we forgot to rediscuss it